pilot/pkg/serviceregistry/kube/controller/ambient/ambientindex.go

```diff
@@ -149,6 +150,10 @@ func New(options Options) Index {
 	// TODO: Should this go ahead and transform the full ns into some intermediary with just the details we care about?
 	Namespaces := krt.NewInformer[*v1.Namespace](options.Client, krt.WithName("Namespaces"))
 
+	EndpointSlices := krt.NewInformerFiltered[*discovery.EndpointSlice](options.Client, kclient.Filter{
```

Comment:
How about Endpoints ? I think they are still supported.

Reply:
There is no code in Istio at all that reads Endpoints anymore, as is the recommendation from Kubernetes. Kubernetes itself has a controlller than syncs Endpoints to EndpointSlices